### ISSUE TITLE
Liste de sujets pour les prochains évenements

### ISSUE THREAD

**Issue by maxdow** [francejs/ToulouseJS#12]
Ici les sujets qui vous intéressent, dont vous voulez qu'on parle ou que vous souhaitez proposer. 

* Redux
* JSPM
* Retour d’expérience angular
* Retour d’expérience React
* Comment tester son application (concepts et techniques adaptés JS ) 
* EcmaScript 6 : présentation des nouveautés 
* Webassembly
* Dataviz ( D3.js .. )
* Js isomorphique ?
* Sujet avec du JS coté Back

**Comment by Torlus** [comment]
JS Everywhere! IoT

2015-09-18 14:50 GMT+02:00 Maxime Warnier <notifications@github.com>:

**Comment by Torlus** [comment]
Remarque, c'est dans une 15aine de jours... Trop court, je ferai ça au
suivant.

2015-09-18 14:51 GMT+02:00 Grégory ESTRADE <gregory.estrade@gmail.com>:

**Comment by maxdow** [comment]
Comme tu le sens .  Dans ce cas je note dans cette liste ( c'était plus pour les sujets après ToulouseJS 8 )

**Comment by cedricdelpoux** [comment]
1. Redux
2. Universal & isomorphic JS
3. Comment faire une appli full JS et faire une api js (NodeJs, express ou autre)

**Comment by enguerran** [comment]
Tu peux ajouter le javascript pour les applications hybrides mobiles :

- apache cordova (webview to appstore)
- ionic framework (most used hybrid framework)
- react-native (le framework de facebook pour les mobiles)
- nativescript

**Comment by simon-previdente** [comment]
Lors de *Paris Web 2015*, *Alvin Berthelot* a donné une conférence sur une **console de jeux web**. J'ai trouvé que c'était plutôt original et sympathique, alors pourquoi pas remettre ça lors d'un ToulouseJS ? :)

**Comment by maxdow** [comment]
@simon-previdente je n'ai pas vu les replay encore mais pourquoi pas. Ça te tente de proposer quelque chose la dessus ? 

@enguerran C'est fait

**Comment by hadrienl** [comment]
Si quelqu'un est calé sur le sujet, j'aimerais bien en savoir plus sur les bénéfices de http2 et la façon dont nous servirons nos webapp dans le futur proche.

**Comment by maxdow** [comment]
@hadrienl ça pourrait même faire l'objet d'un apéro web ping @DirtyF

**Comment by DirtyF** [comment]
J'ai lu quelques trucs sur HTTP2 comme tout le monde et notamment ce que ça change au niveau des bonnes pratiques webperf, il me semble que @dmathieu avait des choses à dire sur HTTP2

**Comment by dmathieu** [comment]
Je ne suis probablement pas la bonne personne pour parler d'http/2 désolé :(

**Comment by hadrienl** [comment]
Laurent Saint-Félix semble s'y connaitre : https://www.youtube.com/watch?v=KqWWThnUM9c

**Comment by DirtyF** [comment]
@anaethelion y'a de la demande pour une petite présentation sur HTTP2, ça te dit ?

**Comment by Anaethelion** [comment]
Si vous avez vraiment envie de m'entendre vous raconter combien c'est fantastique, pourquoi pas. :)

**Comment by hadrienl** [comment]
Si tu arrives à te concentrer sur les avantages que ça apportera à nous autres, dev js, je donne mon vote avec une immense joie !

**Comment by Anaethelion** [comment]
Oh oui y'a plein de trucs dont j'ai pas parlé dans mon petit dej' qui concernent surtout les devs front et qui intéresseront du monde. Notamment le server push et le fait que plusieurs techniques actuelles d'optimisation de perfs sont contre productives en http2.

**Comment by Torlus** [comment]
Javascript of Things!
Faut que je me bouge un peu...
Pas le prochain Toulouse.js mais celui d'après.

2015-10-27 22:21 GMT+01:00 Laurent Saint-Félix <notifications@github.com>:

**Comment by maxdow** [comment]
Le prochain est en principe booké donc ce sera pour celui de décembre ( date à définir ) , et éventuellement http2 oui si tu es partant @Anaethelion

**Comment by Torlus** [comment]
Ah ouais, je veux faire mon talk le même jour que @Anaethelion :)

2015-10-27 22:29 GMT+01:00 Maxime Warnier <notifications@github.com>:

**Comment by maxdow** [comment]
Je finalise le prochain TJS #9 et ensuite on attaque le 10 ,  la date sera fixée la semaine prochaine ;)

**Comment by Inateno** [comment]
Je propose comme sujet "Des application desktop en JS avec nw.js ou Electron" :)
Celui sur une console de jeux web m'intéresse, si vous avez le replay :+1:

**Comment by hadrienl** [comment]
Ça me plait bien ça !

**Comment by cedricdelpoux** [comment]
ça peut être sympa comme sujets ça : 
* Sujet avec du JS coté Back
* JS Everywhere! Io

**Comment by maxdow** [comment]
JS Everywhere c'est un talk prévu .

JS coté back la difficulté c'est de trouver quelqu'un avec un retour d'expérience concret qui dépasse le proto maison ou le tuto openclassroom ;) . La dessus j'aimerai évoquer les problématiques de cluster ( gestion multiproc ), tenue à la charge, intégration avec nginx ou autre, facilités/ contraintes de dev et déploiement vs les autres technos etc

**Comment by Inateno** [comment]
Je peux faire un talk JS côté back, ma plateforme de jeu est online depuis 1 ans ça peut faire un bon post-mortem "NodeJS en prod ça donne quoi"
Y'aura beaucoup de trucs discutable, notamment pas de cluster (choix voulu), partie backend, auth via socket moisie, mongodb sur le long terme, bref de tout :)

Si ça vous intéresse alors why not

**Comment by maxdow** [comment]
Il me semble que tu es la lundi prochain? on en discute avec plaisir :)

**Comment by Inateno** [comment]
yep je suis là (sûrement a la bourre, venant de blagnac faut que je me gare + le boxon du soir bref)

**Comment by MoOx** [comment]
Je peux gerer redux un de ces 4 si ca vous dit

**Comment by maxdow** [comment]
Sur le prochain j'ai bien envie qu'on parle back . Ensuite un petit sujet redux me semble pas mal oui :)

**Comment by Torlus** [comment]
Si on case JSoT, HTTP/2 et un sujet back, ça serait rigolo, ça ferait une
session "font-end forbidden" (ou front-403, avec du front-242 en bande-son,
mais je m'égare...)

**Comment by Inateno** [comment]
Toujours opé pour du back, ça risque d'en faire crier plus d'un par contre ^^

**Comment by enguerran** [comment]
Pourquoi donc ?
ToulouseJS c'est pour parler de Javascript, non ? Dans la veille de Max, on parle souvent de NodeJS. Les cas d'utilisation de javascript évolue et nous avec.

En revanche un _sujet back_ ça me parait un peu vague. Ce serait quoi le sujet ?

**Comment by Inateno** [comment]
c'est NodeJS en production le talk en fait, c'est dans la liste des sujets "Sujet avec du JS coté Back" d'où mon raccourcis vers "back" mais bref ;)

**Comment by enguerran** [comment]
:+1: cool

**Comment by JulienPradet** [comment]
Est-ce que vous pensez que ça peut être intéressant un talk sur [Recompose](https://github.com/acdlite/recompose) et plus généralement sur les HOCs ? Je veux bien faire un pti qqch.

**Comment by MoOx** [comment]
Bonne idée

**Comment by maxdow** [comment]
Absolument @JulienPradet . On en parle à la prochaine session si tu es ok (surement fin octobre )

**Comment by JulienPradet** [comment]
@maxdow Ca marche. Je suis déjà inscrit.
Je serai le timide dans le coin. ;p
